config/locales/en.yml

```diff
@@ -0,0 +1,56 @@
+en:
+  label_redmine_zulip: Zulip
+  zulip_settings_header: Zulip Plugin Configuration
+  zulip_settings_label_url: Zulip API URL
+  zulip_settings_label_email: Zulip Bot Email
+  zulip_settings_label_api_key: Zulip Bot API key
+  field_zulip_stream: Stream name
+  field_zulip_private_messages: Get private notifications
+  field_zulip_subject_issue: Create subjects for issues
+  field_zulip_subject_version: Create subjects for versions
+  zulip_notify_assignment: >
+    %{user} **assigned** issue [%{id}: %{subject}](%{url}) to **you**
+
+
+    ~~~quote
+
+    %{description}
+
+    ~~~
+
+
+    **%{status_label}**: %{status}
+
+
+  zulip_notify_unassignment: >
+    %{user} **unassigned** issue [%{id}: %{subject}](%{url}) from **you**
+
+  zulip_notify_updated: >
+    %{user} **updated** issue [%{id}: %{subject}](%{url})
+
+  zulip_notify_destroyed: >
+    %{user} **deleted** issue **%{id}: %{subject}**
```

Comment:
So, these two are a bit trickier, we can't really end with punctuation here because the `subject` could contain its own trailing punctuation. Is there a way to figure out whether the `subject` already ends in punctuation or not?

Reply:
@eeshangarg it now detects and removes ending dots (`.`) in `subject`